### DISCUSSION TITLE
Multiple Dual Edge TPUs

### DISCUSSION THREAD

**Discussion by RedAnon** [blakeblackshear/frigate#4914] [General]
At what point is another TPU needed?
How many cameras can a single TPU process before it starts lagging?

**Comment by NickM-27** [comment]
You can watch the skipped fps sensor to know if fps is being skipped. The coral inference time as a function of 1000 milliseconds being in a second will tell you how many detections can be run per second. Frigate will usually run multiple detections for a camera on a single frame that has motion. 

So it depends how many cameras you have, their fps, and how much activity they see.
